src/main/java/com/toptrumps/cli/CommandLineController.java

```diff
@@ -76,14 +77,17 @@ private void startNewGame(int numberOfOpponents) {
 
             // == ATTRIBUTE SELECTION ==
 
+
             Attribute selectedAttribute;
             if (activePlayer.isAIPlayer()) {
                 selectedAttribute = ((AIPlayer) activePlayer).selectAttribute();
             } else {
                 selectedAttribute = onRequestSelection(activePlayer.getTopCard());
                 activePlayer.setSelectedAttribute(selectedAttribute);
+
             }
             onAttributeSelected(activePlayer);
+            boolean humanTurn = activePlayer.isAIPlayer();
 
```

Comment:
Why is human turn true when the active player is AI? This seems to be counter intuitive